VM/include/lua.h

```diff
@@ -129,6 +129,7 @@ LUA_API int lua_isstring(lua_State* L, int idx);
 LUA_API int lua_iscfunction(lua_State* L, int idx);
 LUA_API int lua_isLfunction(lua_State* L, int idx);
 LUA_API int lua_isuserdata(lua_State* L, int idx);
+LUA_API int lua_isvector(lua_State* L, int idx);
```

Comment:
It might make more sense for this to be a macro that calls lua_type similar to all other "simple" type checks

Reply:
You're of course right. Somehow I missed those macros. I'll fix it tomorrow.

Reply:
No worries, I fixed this up and switched conformance test to use the new function.